README.md

```
@@ -1,63 +1,91 @@
+これで以下が自動的に行われます:
 
-## 管理している設定ファイル
+- CLI ツールのインストール (Nix)
+- GUI アプリのインストール (Homebrew cask)
+- dotfile の symlink 作成 (home-manager)
+- macOS システム設定の適用
+- Claude Code の MCP サーバー登録・スキルインストール
+
+## 構成
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)